### DISCUSSION TITLE
Usage in preact

### DISCUSSION THREAD

**Discussion by waptik** [Telegram-Mini-Apps/tma.js#202] [Q&A] [answered]
How do i use this library in preact? I see that you support reactjs already.
I'm using[ deno fresh](https://fresh.deno.dev) which uses [preact](https://preactjs.com) under the hood.

**Comment by heyqbnk** [comment, marked as answer]
I am not really sure you can. But at the same time, that last I was working with preact was about 4 years ago. Maybe this topic will be helpful, but not sure:
https://preactjs.com/guide/v10/switching-to-preact

**Reply by waptik** [reply]
Got it! I'll check it out. Thanks
